compiler.rb

```diff
@@ -86,6 +88,12 @@
   opt.on('-d', '--debug', 'Show all debug logs') do |_debug|
     Google::LOGGER.level = Logger::DEBUG
   end
+  opt.on('-c', '--code', 'Generate code only') do
```

Comment:
Would you consider doing this in the opposite direction?  default to true, with flags for "--no-code" and "--no-docs"?  I know it's a nitpick but that feels more consistent to me and more readable in scripts.  Generally speaking a --flag adds behavior unless it starts with --no.

Reply:
@ndmckinley - i think i made the change you were looking for, let me know if i misinterpreted.  thanks!!